src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -288,6 +288,9 @@ def get_payment_context_data(submission: Submission) -> dict[str, Any]:
             "completed": submission.payment_user_has_paid,
             "amount": str(amount),
             "public_order_ids": submission.payments.get_completed_public_order_ids(),
+            "provider_payment_ids": list(
+                submission.payments.values_list("provider_payment_id", flat=True)
+            ),
```

Comment:
I think this one also incorrectly includes the possible failed payment IDs and they should be filtered out, inline with how only the public order IDs of completed payments are used above